codec/codec_test.go

```diff
@@ -0,0 +1,134 @@
+package codec_test
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	"github.com/heetch/felice/codec"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+// checks if codec encodes to the proper output.
```

Comment:
:tulip: The "checks if" at the start of all these test descriptions is redundant.  We know they're checking something.  It's better to just make a plain statement.  Also, this sort of case might not even need a comment at all (I often write them also, but I don't think "it does the right thing" is adding anything much).